packages/go_router/example/lib/on_exit.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Can you write a test for this example, this can just be a simple test to make sure things compile and the primary use case works

Reply:
Added the test here [Test on_exit example](https://github.com/flutter/packages/pull/3709/commits/20b004de2ac1b85dc4cf748807def2ca210f0cb1)